app/src/components/ContributionDetail.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <template v-if="contributions.length > 0">
+    <section v-for="contribution in contributions" :key="contribution.grantName" class="px-4 md:px-12">
+      <div class="py-4 md:py-6 lg:py-8 border-b border-grey-100">
+        <div class="grid grid-cols-12 gap-x-8 gap-y-4 items-center">
+          <!-- grand image + grant description + optional grant round -->
+          <article class="col-span-12 md:col-span-6 lg:col-span-5">
+            <!-- subgrid -->
+            <div class="grid grid-cols-4 items-center gap-x-4">
+              <!-- img -->
+              <div class="hidden lg:block col-span-1">
+                <!-- do not forget to link the figure to the grant ^^ -->
+                <figure class="aspect-w-16 aspect-h-9 shadow-light cursor-pointer">
+                  <img
+                    class="w-full h-full object-center object-cover group-hover:opacity-90"
+                    :src="contribution.grantLogoURI ? contribution.grantLogoURI : '/placeholder_grant.svg'"
+                  />
+                </figure>
+              </div>
+              <!-- text -->
+              <div class="col-span-4 lg:col-span-3">
+                <a href="#" class="link">{{ contribution.grantName }}</a>
+                <!-- if this tx was via a matching round show round -->
+                <br /><span class="mr-2">via</span>
+                <a href="#" class="link">{{ contribution.roundName }}</a>
+                <!-- end if -->
+              </div>
+            </div>
+          </article>
+
+          <!-- donors + amount -->
+          <article class="col-span-12 md:col-span-6 lg:col-span-4">
+            <!-- subgrid -->
+            <div class="flex justify-between items-center">
+              <!-- donors -->
+              <div>
```

Comment:
do we need this extra div ?

Reply:
yes. i need all my code exactly as i did it.   there might be  1-2 unnessesary divs for sure what are super helpfull on another level of architecture and structure and later readability.

Reply:
@thelostone-mc @mds1 in favor of what @scco I am going to leave all html the way it (where comments ask if div is necessary). If that is alright with everyone?

Reply:
Vue's `template` tag is generally what you should use if you need a div just for structure/readability that doesn't need to be rendered in the DOM